### DISCUSSION TITLE
2021 Release

### DISCUSSION THREAD

**Discussion by PatheticMustan** [Prodigy-Hacking/ProdigyMathGameHacking#409] [(D) Developer]
For our 3k member special ([Discord](https://discord.gg/XQDfbfq)), we're planning on releasing a special 2021 release!

#408 

About to add morph hacks, and update "Win Battle" for rune runs

**Comment by ghost** [comment]
Wow that’s amazing! Congrats on almost 3k members (btw my user is Y33T#5079)!

**Comment by PatheticMustan** [comment]
Beta released the 2021 update for WCM...

https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/pull/408/commits/3604d6ecf8dc739f7d767f792c807fbb82c029f5

**Comment by poIarbear** [comment]
natsumi give now even though i don't do this anymore..

give

**Comment by PatheticMustan** [comment]
.... we already released it tho....?
